src/AngularNoNgAttributesSnapshotSerializer.js

```diff
@@ -3,23 +3,51 @@
 const jestDOMElementSerializer = require('pretty-format').plugins.DOMElement;
 
 const attributesToRemovePatterns = ['ng-reflect', '_nghost', '_ngcontent', 'ng-version'];
+const attributesToClean = {
+  class: [/^.*-\w*\d+-\d+$/, /^ng-star-inserted$/], // e.g. "ng-tns-c25-1" or "ng-star-inserted", literally
+  id: [/^.*-\d+$/], // e.g. "mat-input-4", "cdk-step-content-0-0"
+  for: [/^.*-\d+$/], // e.g. "mat-form-field-label-9"
+  'aria-owns': [/^.*-\d+$/], // e.g. "mat-input-4"
+  'aria-labelledby': [/^.*-\d+$/], // e.g. "mat-input-4", "cdk-step-label-0-0"
+  'aria-controls': [/^.*-\d+$/], // e.g. "cdk-step-content-2-0"
+};
```

Comment:
All of the classes do start with `mat`, `cdk` or `ng` and I think we really should take advantage of this.
There might some project using `component-nr25-1` and their class would "randomly" be removed.
As this is a unit test library that usually does not care about styles, I can understand arguments that the "damage" is negligible, but I think it's too aggressive to just remove all classes that end on the pattern.